ocaml/testsuite/tests/typing-float32/types.ml

```diff
@@ -0,0 +1,13 @@
+(* TEST
+   * expect
+*)
+
+type t = float32;;
+[%%expect{|
+type t = float32
+|}];;
+
+type t = float32array;;
+[%%expect{|
+type t = float32array
+|}];;
```

Comment:
I think we should have more tests .  In particular, we should be checking that the compiler does indeed blow up loudly if you try to compile a program with a float32.

Reply:
(Discussed offline)

Reply:
Thanks!